readme.md

```diff
@@ -11,15 +23,15 @@ Once you've installed and activated the plugin, try using [Quill](http://quill.p
 Supports the [full W3C Micropub CR spec](https://www.w3.org/TR/micropub/) as of 2016-10-18, except for the optional media endpoint. Media may be uploaded directly to the wordpress-micropub endpoint as multipart/form-data, or sideloaded from URLs.
 
 
-### WordPress details
+## WordPress details 
 
-#### Filters and hooks
 
+### Filters and hooks 
 Adds two filters:
 
 `before_micropub( $input )`
 
-Called before handling a Micropub request. Returns `$input`, possibly modified.
```

Comment:
odd that the backticks here got lost. any idea why?

Reply:
No, the conversion is done in software. But let me have a look at the converter.

Reply:
ping?